metals/src/main/scala/scala/meta/internal/metals/Compilers.scala

```diff
@@ -418,6 +418,19 @@ class Compilers(
     }
   }.getOrElse(Future.successful(Nil.asJava))
 
+  def inlineEdits(
+      uri: String,
+      range: LspRange,
+      inlineAll: Boolean,
+      token: CancelToken,
+  ): Future[Either[String, List[TextEdit]]] =
+    withPCAndAdjustLsp(uri, range, range.getStart) { (pc, pos, _, adjust) =>
+      pc.inlineValue(CompilerRangeParams.fromPos(pos, token), inlineAll)
+        .asScala
+        .map(_.asScala)
+        .map(_.map(_.asScala.toList))
```

Comment:
```suggestion
        .asScala
        .map(_.asScala.map(adjust.adjustTextEdits))
```
Might need adjusting to work also in worksheets.
Also, we don't need to do `asScala` and then `asJava` in `InlineValueCodeAction`

Reply:
Although it looks like worksheets support might blocked by https://github.com/scalameta/metals/issues/4793